maintainers/maintainer-list.nix

```diff
@@ -1037,6 +1037,12 @@
     githubId = 3465841;
     name = "Boris Sukholitko";
   };
+  bqv = {
+    email = "bqv@fron.io";
+    github = "bqv";
+    githubId = 822863;
+    name = "Tony O";
+  };
```

Comment:
generally this is a separate commit, sorry if I forgot to mention it
```
maintainers: add bqv
```

Reply:
No problem. I've separated this out